### ISSUE TITLE
Orientation of latent space

### ISSUE THREAD

**Issue by GangLiTarheel** [ratschlab/scim#9]
Hi,
I am trying to run SCIM on my own data set. I noticed that in paper it mentions that "adopt a semi-supervised approach
by adding a ‘censored’ label and randomly relabel cells in the training set.". 

May I ask:
1. How to incorporate my own labels in current SCIM framework (which variables corresponds to that)?
2. Did the tutorial (simulation data) utilizes all the labels or just 10% of labels? Could you explain what do you mean by "censored"? Is there a variable to control which labels are used for the orientation?
3. How do you randomly relabel cells for TuPro dataset? I think the labels that you released are the correct ones, right? If we want to reproduce the results in the paper, how should I proceed?

Excellent work! Thank you for your time and efforts!
Best,
Gang

**Comment by stefangstark** [comment]
See demo.ipynb, labels can be loaded from an anndata file using obs column name. censored means that a subset labels are changed to eg a "nan" category
